sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_get_validation_info_from_recognize_custom_forms_async.py

```diff
@@ -0,0 +1,71 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_get_validation_info_from_recognize_custom_forms_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to output the information that will help with manually
+    validating your output from recognize custom forms.
+USAGE:
+    python sample_get_validation_info_from_recognize_custom_forms_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Cognitive Services resource.
+    2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer subscription key
+    3) CUSTOM_TRAINED_SUPERVISED_MODEL_ID - the ID of your custom trained supervised model
+"""
+
+import os
+import asyncio
+
+
+class GetValidationInfoFromRecognizeCustomFormsSampleAsync(object):
+
+    endpoint = os.environ["AZURE_FORM_RECOGNIZER_ENDPOINT"]
+    key = os.environ["AZURE_FORM_RECOGNIZER_KEY"]
+    supervised_model_id = os.environ["CUSTOM_TRAINED_SUPERVISED_MODEL_ID"]
+
+    async def get_validation_info_from_recognize_custom_forms(self):
+        # TODO: this can be used as examples in sphinx
+        from azure.core.credentials import AzureKeyCredential
+        from azure.ai.formrecognizer.aio import FormRecognizerClient
+        form_recognizer_client = FormRecognizerClient(endpoint=self.endpoint, credential=AzureKeyCredential(self.key))
```

Comment:
I'd rather use the context manager syntax
```python
async with FormRecognizerClient(endpoint=self.endpoint, credential=AzureKeyCredential(self.key)) as form_recognizer_client 
```
to avoid the close